worker.go

```diff
@@ -581,15 +582,25 @@ func isNoUpdatesToBePerformedError(err error) bool {
 	return false
 }
 
-func updateIngress(kubeAdapter *kubernetes.Adapter, lb *loadBalancer, problems *problem.List) {
+func updateIngress(ctx context.Context, awsAdapter *aws.Adapter, kubeAdapter *kubernetes.Adapter, lb *loadBalancer, problems *problem.List) {
 	var dnsName string
 	if lb.clusterLocal {
 		dnsName = kubernetes.DefaultClusterLocalDomain
 	} else {
-		// only update ingress if the stack exists and is in a complete state.
+		// do not update ingress if the stack exists and is in a complete state.
```

Comment:
I think this was correct. If you want to negate the statement it should be "do not update ingress if the stack does not exist or is not in a complete state."